Assets/Mirror/Runtime/Transport/Transport.cs

```diff
@@ -125,6 +126,20 @@ public virtual bool Available()
         /// <returns>true if the data was sent</returns>
         public abstract bool ServerSend(int connectionId, int channelId, byte[] data);
 
+        /// <summary>
+        /// Send data to multiple clients at once.
+        /// Calls ServerSend by default. Can be overwritten by transports that can send to multiple
+        /// clients at once more efficiently if needed.
+        /// </summary>
+        public virtual bool ServerSend(IEnumerable<int> connectionIds, int channelId, byte[] data)
```

Comment:
merging this in master is a commitment that we will not break it.   This means we won't be able to change it to arraysegment without a breaking change later on.

Thus this should not be merged in master,  we can keep it in a separate branch until we change it.